### PR TITLE
+

### PR DESCRIPTION
**Pull request by AndriZhok**

### CODE REVIEW

File: taxi/views.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
 from django.shortcuts import render
+from .models import Driver, Manufacturer, Car
```
Comment (Dimosphen1) [inline taxi/views.py:2]: Prefer using absolute imports here and in the other places as well

File: templates/taxi/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Taxi Service Dashboard</h1>
+  <ul>
+    <li>Number of cars: {{ num_cars }}</li>
+    <li>Number of drivers: {{ num_drivers }}</li>
+    <li>Number of manufacturers: {{ num_manufacturers }}</li>
+  </ul>
+{% endblock %}
```
Comment (Dimosphen1) [inline templates/taxi/index.html:10]: Add blank line to the end of the file

File: static/css/styles.css
Comment (Dimosphen1) [inline static/css/styles.css:1]: Add content to this file to demonstrate that static files are served correctly in the project. Attach corresponding screenshots as well

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

**Review by pavlopro — APPROVED** [review]
Good Job!

**Pull request closed without merging**